index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <span ><a class="myLogo" href="#">Abedy</a></span>
+        <nav>
+            <a href="#">
+                <img src="icons/Union.png" alt="Union.png">
+            </a>
+        </nav>
+    </header>  
+    <section class="headline">
+        <h1 class="heading">I’m Abednego<br> Glad to see<br> you!</h1>
+        <main>
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </main>
+        <h2 class="connect">LET'S CONNECT</h2>
+        <div>
+            <ul class="social">
+                <li><a href="#"><img src="icons/Vector.png" alt="twitter_icon"></a></li>
+                <li><a href="#"><img src="icons/Linkedin icon.png" alt="Linkedin_icon"></a></li>
+                <li><a href="#"><img src="icons/Vector (1).png" alt="Medium_icon"></a></li>
+                <li><a href="#"><img src="icons/Vector (2).png" alt="Github_icon"></a></li>
+                <li><a href="#"><img src="icons/Vector (3).png" alt="Chill_icon"></a></li>
+            </ul>
+        </div>    
+    </section>  
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you update your styling by adding the correct spacing the top of your hero text. Make sure you fix as it was not mentioned by the previous reviewer.

| Template design | Your Design |
| ------------------- | -------------------- |
| ![Screenshot from 2023-02-15 18-12-26](https://user-images.githubusercontent.com/83042742/219085728-9419a1eb-a965-4c52-b75b-cf25ac52eb1d.png) |![Screenshot from 2023-02-15 18-06-24](https://user-images.githubusercontent.com/83042742/219086090-80309757-8c87-43d6-afe4-f0f4b8900e5d.png) |